src/main/resources/org/sagebionetworks/web/client/widget/entity/editor/UserSelectorViewImpl.ui.xml

```diff
@@ -6,7 +6,7 @@
 	
 	<b:Modal ui:field="modal" closable="true" dataBackdrop="STATIC" title="Find Synapse User or Team">
 		<b:ModalBody>
-			<bh:Div ui:field="suggestBoxContainer"/>
+			<bh:Div ui:field="suggestBoxContainer" paddingTop="30" paddingBottom="50"/>
```

Comment:
Noticed that if you click Insert->User in the wiki editor, the user selector modal did not have proper space around the textbox.

Reply:
Here it is after the change: <img width="623" alt="Screen Shot 2021-03-31 at 10 41 59 AM" src="https://user-images.githubusercontent.com/1864447/113187574-dde5ff00-920d-11eb-9c09-9ee19f7f54da.png">